src/common/core/constants.js

```diff
@@ -50,6 +50,9 @@ define([], function () {
             CONSTRAINT_COLLISION: 'constraint collision'
         },
 
+        EXPORT_TYPE_FULL_PROJECT: 'full',
```

Comment:
Isn't 'project' more suiting? Like 'library' and 'project', vs 'library' and 'full'?